apps/backoffice-v2/src/pages/Entity/components/Case/case-action-variants/WebsiteMonitoringCaseActions/hooks/useWebsiteMonitoringCaseActions/useWebsiteMonitoringCaseActions.ts

```diff
@@ -0,0 +1,70 @@
+import { IUseActions } from '@/pages/Entity/components/Case/hooks/useCaseActionsLogic/interfaces';
+import { useCaseActionsLogic } from '@/pages/Entity/components/Case/hooks/useCaseActionsLogic/useCaseActionsLogic';
+import { StateTag } from '@ballerine/common';
+import { useCallback, useState } from 'react';
+
+export interface IResolutionActionsFormValues {
+  action: string;
+  comment: string;
+}
+
+export const useWebsiteMonitoringCaseActions = ({ workflowId: id, fullName }: IUseActions) => {
+  const {
+    tag,
+    assignedUser,
+    authenticatedUser,
+    isLoadingCase,
+    assignees,
+    isLoading,
+    workflow,
+    isActionButtonDisabled,
+    onMutateRejectEntity,
+    onMutateApproveEntity,
+    onMutateAssignWorkflow,
+  } = useCaseActionsLogic({ workflowId: id, fullName });
+
+  const [formValues, setValues] = useState<IResolutionActionsFormValues>({
```

Comment:
The type is already inferred by the default value.
```suggestion
  const [formValues, setValues] = useState({
```

Reply:
P3